### ISSUE TITLE
Problem with cell - one of them is empty

### ISSUE THREAD

**Issue by pionas** [vandeseer/easytable#50]
My code:
```
        final TableBuilder tableBuilder = Table.builder()
                .addColumnsOfWidth(35, 92, 50, 50, 72, 240)
                .fontSize(8)
                .font(HELVETICA)
                .borderColor(Color.BLACK);

        tableBuilder.addRow(Row.builder()
                .add(TextCell.builder().text("XXXXXXXXX")
                        .colSpan(4)
                        .lineSpacing(1f)
                        .borderWidthTop(1)
                        .textColor(Color.BLACK)
                        .backgroundColor(Color.RED)
                        .fontSize(14)
                        .font(PDType1Font.HELVETICA_BOLD)
                        .verticalAlignment(VerticalAlignment.MIDDLE)
                        .horizontalAlignment(CENTER)
                        .borderWidth(1)
                        .build())
                .add(TextCell.builder().text("YYY YYY")
                        .colSpan(2)
                        .textColor(Color.BLACK)
                        .backgroundColor(Color.RED)
                        .fontSize(11)
                        .font(PDType1Font.HELVETICA_BOLD)
                        .verticalAlignment(VerticalAlignment.MIDDLE)
                        .horizontalAlignment(CENTER)
                        .borderWidth(1)
                        .build())
                .horizontalAlignment(LEFT)
                .build());

        tableBuilder.addRow(Row.builder()
                .add(TextCell.builder().text("#")
                        .rowSpan(3)
                        .lineSpacing(1f)
                        .borderWidthTop(1)
                        .textColor(Color.BLACK)
                        .backgroundColor(GRAY_LIGHT_3)
                        .fontSize(8)
                        .font(PDType1Font.HELVETICA_BOLD)
                        .verticalAlignment(VerticalAlignment.MIDDLE)
                        .horizontalAlignment(CENTER)
                        .borderWidth(1)
                        .build())
                .add(TextCell.builder().text("CCC")
                        .rowSpan(3)
                        .textColor(Color.BLACK)
                        .backgroundColor(GRAY_LIGHT_3)
                        .fontSize(8)
                        .font(PDType1Font.HELVETICA_BOLD)
                        .verticalAlignment(VerticalAlignment.MIDDLE)
                        .horizontalAlignment(CENTER)
                        .borderWidth(1)
                        .build())
                .add(TextCell.builder().text("QWERTY")
                        .colSpan(3)
                        .textColor(Color.BLACK)
                        .backgroundColor(Color.YELLOW)
                        .fontSize(10)
                        .font(PDType1Font.HELVETICA_BOLD)
                        .verticalAlignment(VerticalAlignment.MIDDLE)
                        .horizontalAlignment(CENTER)
                        .borderWidth(1)
                        .build())
                .add(TextCell.builder().text("John Black")
                        .textColor(Color.BLACK)
                        .backgroundColor(Color.YELLOW)
                        .fontSize(8)
                        .verticalAlignment(VerticalAlignment.MIDDLE)
                        .horizontalAlignment(CENTER)
                        .borderWidth(1)
                        .build())
                .horizontalAlignment(LEFT)
                .build());

        tableBuilder.addRow(Row.builder()
                .add(TextCell.builder().text("QQQQ")
                        .colSpan(3)
                        .lineSpacing(1f)
                        .borderWidthTop(1)
                        .textColor(Color.BLACK)
                        .backgroundColor(GRAY_LIGHT_3)
                        .fontSize(8)
                        .font(PDType1Font.HELVETICA_BOLD)
                        .verticalAlignment(VerticalAlignment.MIDDLE)
                        .horizontalAlignment(CENTER)
                        .borderWidth(1)
                        .build())
                .add(TextCell.builder().text("ZZZZZ")
                        .rowSpan(2)
                        .lineSpacing(1f)
                        .borderWidthTop(1)
                        .textColor(Color.BLACK)
                        .backgroundColor(GRAY_LIGHT_3)
                        .fontSize(8)
                        .font(PDType1Font.HELVETICA_BOLD)
                        .verticalAlignment(VerticalAlignment.MIDDLE)
                        .horizontalAlignment(CENTER)
                        .borderWidth(1)
                        .build())
                .horizontalAlignment(LEFT)
                .build());

        tableBuilder.addRow(Row.builder()
                .add(TextCell.builder().text("X")
                        .lineSpacing(1f)
                        .borderWidthTop(1)
                        .textColor(Color.BLACK)
                        .backgroundColor(GRAY_LIGHT_3)
                        .fontSize(8)
                        .font(HELVETICA)
                        .verticalAlignment(VerticalAlignment.MIDDLE)
                        .horizontalAlignment(CENTER)
                        .borderWidth(1)
                        .build())
                .add(TextCell.builder().text("Y")
                        .lineSpacing(1f)
                        .borderWidthTop(1)
                        .textColor(Color.BLACK)
                        .backgroundColor(GRAY_LIGHT_3)
                        .fontSize(8)
                        .font(HELVETICA)
                        .verticalAlignment(VerticalAlignment.MIDDLE)
                        .horizontalAlignment(CENTER)
                        .borderWidth(1)
                        .build())
                .add(TextCell.builder().text("LLLLLLLLLL")
                        .lineSpacing(1f)
                        .borderWidthTop(1)
                        .textColor(Color.BLACK)
                        .backgroundColor(GRAY_LIGHT_3)
                        .fontSize(8)
                        .font(HELVETICA)
                        .verticalAlignment(VerticalAlignment.MIDDLE)
                        .horizontalAlignment(CENTER)
                        .borderWidth(1)
                        .build())
                .horizontalAlignment(LEFT)
                .build());

        return tableBuilder.build();
```

**Comment by vandeseer** [comment]
Thanks for reporting, I will have a look.

**Comment by pionas** [comment]
In my opinion the problem is with skipped columns... -> https://github.com/vandeseer/easytable/blob/master/src/main/java/org/vandeseer/easytable/structure/Table.java#L121

I must add this code:
```
if (cell instanceof TextCell)
{
    if ("ZZZZZ".contains(((TextCell) cell).getText()))
    {
        skipped = 2;
    }
}
```
before line 121 and it work but this is not the solution

**Comment by pionas** [comment]
I find the solution.
https://github.com/vandeseer/easytable/blob/master/src/main/java/org/vandeseer/easytable/structure/Table.java#L112-L114

must change to:
```
for (int i = 0; i < cell.getRowSpan(); i++)
{
    if (rowSpanCells.contains(new Point(rows.size(), i)))
    {
        skipped++;
    }
}
```

**Comment by vandeseer** [comment]
Unfortunately this does not yet fix it. Try to run the test class `ExcelLikeExampleTest`. You will see that the second table is not rendered correctly now. But I think your change is pointing in the right direction :wink:

**Comment by pionas** [comment]
I think you can't skip first row's cell because the first row's cell can be the second or third element for another row's cell...

**Comment by vandeseer** [comment]
Should be fixed in `develop` branch now.

**Comment by pionas** [comment]
Now it's okey :)
Good job :smile:

**Comment by vandeseer** [comment]
Fixed in release 0.5.2
